### PR TITLE
Rework Project structure

### PR DESCRIPTION
**Pull request by Hennzau**
Few changes:

### Structure

- I added SO100 arm folder structure
- I created a node_hub with useful nodes that can be used by every arm
- I changed the name `alexk_low_cost_robot` into `alexk_lcr`

- I added a client structure for each motor type, and an interpolate node for each arm to easily send data between arms (alexk_lcr <-> so100)

### Code

- Inside node_hub there is 2 client nodes: dynamixel client and feetech client. Those nodes can be used to make a link between other dora nodes and a bus of dynamixel motors / feetech motors.
- Interpolate node for alexk lcr to alexk_lcr
- Inside node_hub: camera node and interface node to easily render images and text + manage keyboard

### CODE REVIEW

File: so100/graphs/bi_teleop_with_ak_lcr.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+nodes:
+  - id: lcr_left_leader
+    custom:
+      source: python
+      args: ../../node_hub/dynamixel_client/client.py
```
Comment (haixuanTao) [inline so100/graphs/bi_teleop_with_ak_lcr.yml:5]: Could be better to just have:
`path: ../../node_hub/dynamixel_client/client.py`

As python might be called python3 in some machine.

**Review by haixuanTao — APPROVED** [review]
Looks great overall!

**Comment by Hennzau** [comment]
PR finished, I will move to another subject. @haixuanTao feel free to merge it when you're good with it

**Review by haixuanTao — APPROVED** [review]

**Comment by haixuanTao** [comment]
I think this looks awesome! Thanks a lot! We will try it out this week in China ☺️

**Pull request merged**